unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/PointerEvents/UIPointerEventsHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using DCL.Controllers;
+using DCL.ECS7.InternalComponents;
+using DCL.ECSComponents.UIAbstractElements;
+using DCL.ECSRuntime;
+using DCL.Models;
+
+namespace DCL.ECSComponents
+{
+    /// <summary>
+    /// Adds interactivity to UI components
+    /// </summary>
+    public class UIPointerEventsHandler : UIElementHandlerBase, IECSComponentHandler<PBPointerEvents>
```

Comment:
we are, strongly, trying to avoid any kind on inheritance on the sdk7 code, specially regarding components handlers